src/Compilers/CSharp/Portable/Lowering/MethodToClassRewriter.cs

```diff
@@ -721,6 +721,17 @@ internal override void AddSynthesizedAttributes(ModuleCompilationState compilati
 
                 AddSynthesizedAttribute(ref attributes, this.DeclaringCompilation.TrySynthesizeAttribute(WellKnownMember.System_Diagnostics_DebuggerHiddenAttribute__ctor));
             }
+
+            internal override TypeSymbol IteratorElementType
+            {
+                get
+                {
+                    // BaseMethodWrapperSymbol should not be rewritten by the IteratorRewriter
+                    // https://github.com/dotnet/roslyn/issues/11649
```

Comment:
As code evolves (especially file moves), it can be quite difficult to trace down the original commit that added a line.
I like including issue links.